deepinv/training/trainer.py

```diff
@@ -137,7 +138,7 @@ def get_samples_offline(self, iterators, g):
     :param None, deepinv.physics.generator.PhysicsGenerator physics_generator: Optional physics generator for generating
         the physics operators. If not None, the physics operators are randomly sampled at each iteration using the generator.
         Should be used in conjunction with ``online_measurements=True``. Also see ``loop_physics_generator``.
-    :param deepinv.loss.Loss, list[deepinv.loss.Loss] metrics: Metric or list of metrics used for evaluating the model.
+    :param deepinv.loss.metric.Metric, list[deepinv.loss.metric.Metric] metrics: Metric or list of metrics used for evaluating the model.
```

Comment:
can we have deepinv.loss.Metric? otherwise I feel is a bit verbose

Reply:
You can either import from `deepinv.loss.Metric`, `deepinv.metric.Metric`, or `deepinv.loss.metric.Metric`. These are all tested in the `test_metrics` tests. This means in most cases it remains backwards compatible.